receiver/statsdreceiver/protocol/statsd_parser.go

```diff
@@ -122,6 +145,18 @@ func (p *StatsDParser) Aggregate(line string) error {
 			parsedMetric.intvalue = parsedMetric.intvalue + savedValue
 			p.counters[parsedMetric.description] = buildCounterMetric(parsedMetric, timeNowFunc())
 		}
+
+	case "h":
+		switch p.observeHistogram {
+		case "gauge":
+			p.timersAndDistributions = append(p.timersAndDistributions, buildGaugeMetric(parsedMetric, timeNowFunc()))
+		}
+
+	case "ms":
```

Comment:
Scattered usage of "timing", "timer", "histogram", "ms", "h", etc strings in multiple places in the code is fragile. It will be better to define these as consts, give them proper names and use everywhere.

Reply:
Good point, will revise